SpokestackTests/NLUTensorflowSlotParserTest.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  NLUTensorflowSlotParserTest.swift
+//  SpokestackTests
+//
+//  Created by Noel Weichbrodt on 2/25/20.
+//  Copyright © 2020 Spokestack, Inc. All rights reserved.
+//
+
+import Foundation
+import Spokestack
+import XCTest
+
+class NLUTensorflowSlotParserTest: XCTestCase {
+    
+    func testParse() {
+        let config = SpeechConfiguration()
+        config.nluVocabularyPath = SharedTestMocks.createVocabularyPath()
+        let parser = try! NLUTensorflowSlotParser(configuration: config, inputMaxTokenLength: 100)
+        let metaData = createMetadata().data(using: .utf8)
+        let metadata = try! JSONDecoder().decode(NLUTensorflowMetadata.self, from: metaData!)
+        
+        // selset parse
+        let taggedInputLocation = zip(["b_location"],["kitchen"])
+        let parsedSelset = try! parser.parse(taggedInput: taggedInputLocation, intent: metadata.intents.filter({ $0.name == "request.lights.deactivate" }).first!)
```

Comment:
This file is sort of integration/component test-shaped in general (you're exercising JSON deserialization as well as the actual slot parsing)...but to help readability, these intents could be pulled out into their own variables.